test/verify/check-packages

```diff
@@ -61,12 +61,7 @@ class TestPackages(MachineCase):
         self.login_and_go("/playground/pkgs")
 
         b.switch_to_top()
-        if b.cdp.mobile:
-            b.click("#nav-system-item")
         b.wait_visible("#nav-system li:contains(Terminal)")
```

Comment:
Note, I factorized this a bit in #16884, so these PRs will collide. Let's see who is first :grin: